src/Tempest/Router/src/Request.php

```diff
@@ -17,6 +17,10 @@ interface Request
         get;
     }
 
+    public ?string $raw {
```

Comment:
Should we also allow streams for large blobs of content?

Reply:
I don't think so, but if the need comes, we can improve later

Reply:
True-ish. We'd need to change the interface, which would be a breaking change. But I'm fine merging this in for now.